buildscript.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+echo "generating specfile"
+python3 gen_spec.py $1
```

Comment:
I think that we should always use `debug` run in CI so our log messages will have all possible info.